src/Microsoft.Azure.WebJobs.Host/Hosting/DrainModeManager.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Listeners;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    internal class DrainModeManager : IDrainModeManager
+    {
+        private readonly ILoggerFactory _loggerFactory;
+        private readonly ILogger _logger;
+
+        public DrainModeManager(ILoggerFactory loggerFactory = null)
```

Comment:
Use an `ILoggerFactory<DrainModeManager>` here.

Reply:
Why do we need the factory? Can't you just use `ILogger<DrainModeManager>` here?

Reply:
The comment was also marked as resolved but I don't see the change here. Can you please just update to use ILogger<T>

Reply:
I remember making that change. I am not sure where i missed the commit. Will make the change again.

Reply:
Ok. this is now done in the new commit.
https://github.com/Azure/azure-webjobs-sdk/pull/2526/files#diff-ca6075feb78589c3cff858c1176209c0R18-R27